packages/vanilla-components/src/components/global-container/global-container.tsx

```diff
@@ -7,19 +7,23 @@ import { css } from 'emotion';
 })
 export class GlobalContainer {
   @Prop() background: string;
+  @Prop() loadingcolor: string;
   @Prop() fontfamily: string;
   @Prop() maxwidth: string;
-  @Prop() poweredby: boolean = true;
-
-  LoadingState() {
+  @Prop() poweredby: boolean;
+  
+  LoadingState(props) {
+    const style = css`
+      background-color: ${props.loadingcolor || '#439B76'};
```

Comment:
Should this default loading color go in contentful? I think this is the only prop that has a fall back default color in vanilla components. Is loading state a special case?

Reply:
I wasn't sure about backwards compatibility here. Current widgets won't have loadingcolor set, but then if vanilla components updates then they'd have loading color appear blank in the form.

Reply:
I like the idea of having defaults here, for when the user doesn't set values in the web components.